machine_learning/k_means_clust.py

```diff
@@ -31,8 +31,8 @@
             record_heterogeneity=heterogeneity,
             verbose=True # whether to print logs in console or not.(default=False)
             )
-  4. Plot the loss function, hetrogeneity values for every iteration saved in
-     hetrogeneity list.
+  4. Plot the loss function, and heterogeneity values for every iteration saved in
```

Comment:
```suggestion
  4. Plot the loss function and heterogeneity values for every iteration saved in
```

Reply:
plt.figure()
plt.plot(loss_list, label='Loss')
plt.xlabel('Iteration')
plt.ylabel('Loss')
plt.title('Loss Function')
plt.legend()
plt.show()
you can try these